packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -42,6 +42,10 @@ final class ParallelFileProcessor
      * @var int
      */
     private const SYSTEM_ERROR_LIMIT = 50;
+    /**
+     * @var int
+     */
+    private const MAX_JOBS_PER_WORKER = 10;
```

Comment:
looking at the RAM consumption I think 10 is a good value. I was wondering why the parallel job size is with 20 pretty big in contrast.

we could play with those numbers of course.

adjusting this to 10 I could run rector across mautic with

```
    $rectorConfig->paths([
        __DIR__.'/app/',
        __DIR__.'/plugins',
    ]);
```

with peaks of 16 GB RAM

Reply:
could you comment what is the different between `MAX_JOBS_PER_WORKER` vs `$jobSize` for future reference? 

It defined at as 3rd arg and param at:

https://github.com/rectorphp/rector-src/blob/9f29bf8434e78a86efe97a08ef3e84c8db2f4a94/config/config.php#L16-L16

and

https://github.com/rectorphp/rector-src/blob/9f29bf8434e78a86efe97a08ef3e84c8db2f4a94/packages/Config/RectorConfig.php#L62